kobo/apps/accounts/adapter.py

```diff
@@ -92,3 +92,9 @@ def set_password(self, user, password):
             )
             user.set_password(password)
             user.save()
+
+    def login(self, request, user):
+        # Override django-allauth login method to use specified authentication backend
+        super().login(request, user)
+        user.backend = settings.AUTHENTICATION_BACKENDS[0]
+        login(request, user, backend=user.backend)
```

Comment:
Please reorder alphabetically the methods in that class (even if `is_open_for_signup` is not part of your PR).